lib/ostruct.rb

```diff
@@ -285,7 +285,9 @@ def dig(name, *names)
 
   #
   # Removes the named field from the object. Returns the value that the field
-  # contained if it was defined.
+  # contained if it was defined, else a NameError is raised.
```

Comment:
I'd phrase this differently, we have to use "block" here. Maybe "If the field is not defined, the result of the block is returned, or a NameError is raised if no block was given."

Reply:
Apologies for the delayed responses. 
Does the next sentence not cover the optional blocks well enough?

```
# Optionally, provide a value that will be returned in the event the
# deleting field does not exist.
```

To ensure blocks are mentioned it could be changed to:
```
  # Optionally, provide a block that will be executed in the event the
  # deleting field does not exist.
```

or perhaps the entire description could read as such:
```
  # Removes the named field from the object and returns the value that the field
  # contained if it was defined. You may optionally provide a block. 
  # If the field is not defined, the result of the block is returned, 
  # or a NameError is raised if no block was given.
```